spec/controllers/concerns/breadcrumbable_spec.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+shared_examples_for "an origin page" do |page|
```

Comment:
I think this sort of thing is better actually in the spec, even if it's not as DRY. Much easier to look at the spec and see the behaviour without jumping around. That might just be me mind, just sayin'.

Reply:
i kinda agree, moving them into the specs.